inference-engine/src/inference_engine/saturated_cast.hpp

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
```

Comment:
can we put this functionality to existing `precision_utils.hpp` instead of creation a new file?

Reply:
Done. Please take a look if I did all right.